### DISCUSSION TITLE
intel silicon macos 12.7 chatgpt sidebar button 太靠近左上角三个按钮

### DISCUSSION THREAD

**Discussion by Leopold94941** [tw93/Pake#814] [General]
<img width="119" alt="Screen Shot 2024-10-10 at 12 56 43" src="https://github.com/user-attachments/assets/f1a8ed60-9700-48dc-b3f1-ace8a17d3e53">

**Comment by nmb4** [comment]
that happens because the website doesn't know the window control buttons are there.
there are three options:
1. get the official ChatGPT Desktop App, no need to create one off the website actually
2. disable `—hide-title-bar` if you still want to use it
3. customize the css of the button like i did for [my t3.chat build](https://github.com/NQMVD/desktop-web-apps), but that requires cloning the repo and manually setting up everything...

**the easiest**: just get the official ChatGPT Desktop App.
